nixos/modules/services/misc/paperless.nix

```diff
@@ -0,0 +1,155 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.paperless;
+in
+{
+  options.services.paperless = {
+    enable = mkEnableOption "Paperless";
+
+    dataDir = mkOption {
+      type = types.str;
+      default = "/var/lib/paperless";
+      description = "Directory to store the Paperless data.";
+    };
+
+    consumptionDir = mkOption {
+      type = with types; nullOr str;
+      default = null;
```

Comment:
Set it to the default value mentioned in the description and remove the if-clause below.

Reply:
Good catch! Fixed.